src/components/BounceRuleDetailed/Modals/ChangeModal/index.jsx

```diff
@@ -13,6 +13,11 @@ import { Row } from "../../../Row";
 import { WriteSelectors } from "../../selectors";
 import "./index.scss";
 
+const shouldDisplay = info => info || "N/A";
+
+const displayUnixAsTime = unixTime =>
```

Comment:
nice pulling this into a function. usually if you're reusing a function like this in multiple files, you can stick in a utilities file and then import the function whereever you're using it to avoid duplication